syntax = "proto3";

package com.mycompany.eosutils;

service BlockInfo {
  rpc FetchBlockInfo (FetchRequest) returns (FetchReply);
}

message FetchRequest {
  string address = 1;
}

message FetchReply {
  uint64 balance_photons = 1;
  string address = 2;
}
